src/app/config.js

```diff
@@ -240,7 +240,19 @@ define([
             }
         },
 
-        referenceLayerOpacity: 0.75
+        referenceLayerOpacity: 0.75,
+
+        getGeometryTypeFromCategory: function (category) {
```

Comment:
this `getGeometryTypeFromCategory` method should probably be put in a separate module that imports the config file. Otherwise this config will get out of control and not be a config anymore.